src/IdentityManager/WebApi/Extensions/AutoMapperConfigurationExtension.cs

```diff
@@ -1,19 +1,29 @@
-﻿using IdentityManager.API;
+﻿using AutoMapperBuilder.Extensions.DependencyInjection;
+using IdentityManager.API;
 using IdentityManager.DataService.Mappers;
+using IdentityManager.WebApi.Mappers;
+using Microsoft.AspNetCore.Identity;
 using Microsoft.Extensions.DependencyInjection;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Travely.IdentityManager.Repository.Abstractions.Entities;
 
 namespace IdentityManager.WebApi.Extensions
 {
     public static class AutoMapperConfigurationExtension
     {
         public static void ConfigureAutoMapper(this IServiceCollection services)
         {
-            
             services.AddAutoMapper(typeof(PersistGrantProfile));
+            services.AddAutoMapper(typeof(UserProfile));
+            services.AddAutoMapperBuilder(builder =>
+            {
+                builder.Profiles.Add(new UserProfile(services.BuildServiceProvider().GetRequiredService<IPasswordHasher<User>>()));
```

Comment:
It is bad building ServiceProvider manually. Better approach https://stackoverflow.com/a/44901316/2572312

Correct approach https://stackoverflow.com/a/49627570/2572312